### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by ipohosov** [entropyxyz/community#140] [Get Test Funds] [answered]
### Address

5F4a4ATz3RSp1kCXP4sPz1pJnpGqUFZQvNqstEkcj6DhBtJN

**Comment by johnnymatthews** [comment, marked as answer]
Test tokens sent. Just so you know, there's a faster, easier way to get Entropy test tokens: https://docs.entropy.xyz/guides/get-funds/
